.github/workflows/publish-dist.yml

```diff
@@ -14,10 +14,10 @@ jobs:
         # 2.14.0; if you update this, audit the diff and ensure that it does not leak/abuse secrets
         uses: dawidd6/action-download-artifact@b9571484721e8187f1fd08147b497129f8972c74
         with:
-          name: dist
+          name: webpack-jumpstart
           workflow: build-dist
           run_id: ${{ github.event.workflow_run.id }}
-          path: dist-unpack
+          path: artifact
```

Comment:
naming nitpick: please set the path to webpack-jumpstart to reflect the tar and artifact name.

Reply:
I prefer the name artifact, in fact.  I find it to be absolutely unambiguous as a directory name in a job that downloads an artifact.  Too many things have the same name, otherwise, and it's harder to read and understand what's what.

I used the same thing in the `/npm install` workflow.

Reply:
I see -- this is a directory *containing* webpack-jumpstart.tar.